rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -85,6 +89,7 @@ docker run --name $docker_name \
   --security-opt apparmor:unconfined \
   --volume /var/drivers/nvidia/current:/usr/local/nvidia:ro \
   --volume /tmp/pai-root/alive/$APP_ID:/alive \
+  --label GPU_NO=$gpu_no \
```

Comment:
suggest follow previous style. refine GPU_NO to PAI_TASK_GPU_IDS

Reply:
PAI only sees gpu from 0 to `PAI_GPU_NUM - 1`, don't you need gpu on host?

Reply:
Only will use this docker label gpu id and job/task info to join with custom exporter's gpu metrics. Have question. exporter will use nvidia-smi's gpu id series, need confirm with you docker label's id is consist with nvidia-smi's gpu id sequence?

Reply:
`nvidia-smi` from host or docker get different results. `GPU_NO` is for gpus on the host.